transport-native-unix-common/src/main/java/io/netty/channel/unix/Errors.java

```diff
@@ -100,6 +101,23 @@ int expectedErr() {
         }
     }
 
+    /**
+     * <strong>Internal usage only!</strong>
+     */
+    public static final class NativeChannelException extends ChannelException {
+        private static final long serialVersionUID = 8222160204268655523L;
+      private final int expectedErr;
```

Comment:
Move `expectedErr` in line with the rest of fields

Reply:
Whoops, I guess I haven't set up IntelliJ on this computer yet!